### DISCUSSION TITLE
Welcome to Awesome-Algorand Discussions!

### DISCUSSION THREAD

**Discussion by aorumbayev** [awesome-algorand/awesome-algorand#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of Algorand community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community and what resource do you want to contribute to the list.

**Comment by Jwaters90** [comment]
Algore Polaris Pollux

**Reply by aorumbayev** [reply]
Hey thanks for joining the discussions thread. Can you clarify on what those titles mean - are those projects on algorand ? If you'd like to propose a new record feel free to open the pr 🙂

**Reply by Jwaters90** [reply]
It is more related to the stars, or derived from the words, names and things relating to the stars or heavens

**Reply by Jwaters90** [reply]
Algor Pollux Polaris
